lib/server/prometheus-middleware.ts

```diff
@@ -1,9 +1,7 @@
 import * as promBundle from 'express-prom-bundle';
-import * as prom from 'prom-client';
 import { RequestHandler } from 'express';
 
 export const applyPrometheusMiddleware = (): RequestHandler => {
-  prom.register.clear();
```

Comment:
@ipsi, if we don't clear and call `applyPrometheusMiddleware` twice, will the custom metrics be added only one time?